runtime/darwinia/src/migrations.rs

```diff
@@ -23,13 +23,6 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> Weight {
-	use fp_storage::{EthereumStorageSchema, PALLET_ETHEREUM_SCHEMA};
```

Comment:
We shouldn't delete this migration. Darwinia runtime has not upgraded to `0.12.3` yet.

Reply:
This is OK only if we use the 1230 spec version for runtime upgrade instead of 1231, please make that clear.